config/builds.js

```diff
@@ -37,7 +37,9 @@ const builds = buildConfigs
   })
   .map(buildConfig => {
     const name = buildConfig.name || '';
-    const env = buildConfig.env || {};
+    const env = buildConfig.env || {
+      SKU_TENANT: args.tenant || ''
```

Comment:
Yeah, it makes sense to me that your environment variables would be added to the default, and you could potentially override too.